integration-tests/client/chainlink.go

```diff
@@ -743,6 +743,56 @@ func (c *Chainlink) ImportVRFKey(vrfExportKey *VRFExportKey) (*VRFKey, *http.Res
 	return vrfKey, resp.RawResponse, err
 }
 
+// MustCreateDkgSignKey creates a DKG Sign key on the Chainlink node
+// and returns error if the request is unsuccessful
+func (c *Chainlink) MustCreateDkgSignKey() (*DKGSignKey, error) {
+	dkgSignKey := &DKGSignKey{}
+	log.Info().Str("Node URL", c.Config.URL).Msg("Creating DKG Sign Key")
+	resp, err := c.APIClient.R().
+		SetResult(dkgSignKey).
+		Post("/v2/keys/dkgsign")
+	if err == nil {
+		err = VerifyStatusCode(resp.StatusCode(), http.StatusOK)
+	}
+	return dkgSignKey, err
+}
+
+// MustCreateDkgEncryptKey creates a DKG Encrypt key on the Chainlink node
+// and returns error if the request is unsuccessful
+func (c *Chainlink) MustCreateDkgEncryptKey() (*DKGEncryptKey, error) {
```

Comment:
Naming: `MustCreateDKGEncryptKey`.

Reply:
answered, waiting for reply from SET team

Reply:
Created refactoring ticket to address this - https://smartcontract-it.atlassian.net/browse/QA-1106